### PR TITLE
Feature more spec errors

### PR DESCRIPTION
**Pull request by rylandg**
Tested with the spec...

### CODE REVIEW

File: test/spec.yml [hunk 1]
```diff
@@ -427,6 +427,30 @@
...
+           exports.handler = async (body, req) => {
+             const msleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+             await msleep(100000);
+           };
+           EOL
+    -   in: bn deploy timeoutforsure
+    -   in: bn invoke timeoutforsure
+        err: |-
+            <html>
+            <head><title>504 Gateway Time-out</title></head>
```
Comment (michaeladda) [inline test/spec.yml]: please open issues for all these unfriendly errors you encounter (these are generated by libra for example)

File: test/test.js [hunk 2]
```diff
@@ -70,6 +70,10 @@ function limitedTest(name, testFn) {
   return test(name, async t => limiter(async () => testFn(t)));
 }
 
+function stripText(origText) {
+  return strip(origText.split('\r').join('').slice(0, -1));
```
Comment (michaeladda) [inline test/test.js]: why split and splice and not a simple trimming `replace(/\r+/g, '')`?

File: test/spec.yml (hunk 1 above)
Comment (rylandg) [reply test/spec.yml]: I know it's a Libra error...

File: test/test.js (hunk 2 above)
Comment (rylandg) [reply test/test.js]: Why use regex when not needed?

Comment (michaeladda) [reply test/test.js]: because replace is more readable than split & join?
(it's also faster, but this is not a real consideration here)

**Pull request merged**